inc/sainexthopgroup.h

```diff
@@ -41,11 +41,45 @@ typedef enum _sai_next_hop_group_type_t
     /** Next hop group is ECMP */
     SAI_NEXT_HOP_GROUP_TYPE_ECMP,
 
+    /** Next hop protection group.  Contains primary and backup next hops. */
+    SAI_NEXT_HOP_GROUP_TYPE_PROTECTION,
+
     /* Other types of next hop group to be defined in the future, e.g., WCMP */
 
 } sai_next_hop_group_type_t;
 
 /**
+ * @brief No switchover triggers supported
+ */
+#define SAI_NEXT_HOP_SWITCHOVER_TRIGGER_NULL                         0x00000000
```

Comment:
Please declare those as ENUM and not as define, inside sdk vendor can retlanslate those to uint8 if needed, but sai interface will be clear